### PR TITLE
Add backward incompatible change to release note

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

### CODE REVIEW

**Comment by sm1990** [comment]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -93,3 +93,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
 
 - Removed the previously deprecated `nats:Message` record. Consequently, corresponding APIs no longer accommodate this record. Users are advised to transition to utilizing subtypes of `nats:AnydataMessage` for continued functionality.
 
+- A bug which resulted the rest paramter of a function is not being final, has been fixed.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:96]:
```suggestion
- A bug which resulted in the rest parameter of a function not being considered final has been fixed.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -93,3 +93,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
...
+
+    ```ballerina
+        function sum(int i, int... j) {
+            j = [1, 2, 3]; // error: cannot assign a value to function argument 'j'
+        }
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:101]: Fix indentation.

**Comment by MaryamZi** [comment]
@LakshanWeerasinghe please resolve the conflict too.

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**